dask_expr/_collection.py

```diff
@@ -2528,6 +2528,13 @@ def pivot_table(df, index, columns, values, aggfunc="mean"):
     ):
         raise ValueError("'values' must refer to an existing column or columns")
 
+    available_aggfuncs = ["mean", "sum", "count", "first", "last"]
+
+    if not is_scalar(aggfunc) or aggfunc not in available_aggfuncs:
+        raise ValueError(
+            "aggfunc must be either " + ", ".join(f"'{x}'" for x in available_aggfuncs)
+        )
```

Comment:
```suggestion
        raise ValueError(
            f"aggfunc must be a scalar or one of: {', '.join(available_aggfuncs)}"
        )
```
mostly b/c I read "either" to mean one of two.

Reply:
The correction is not completely correct, lists are generally not allowed but we also only allow the list that is hardcoded.

This would need an update in dask/dask as well to handle the testing suite